README.md

```diff
@@ -126,14 +126,36 @@ saveRDS(mod, temp_file)
 mod2 <- readRDS(temp_file)
 
 compare(mod, mod2)
-#> `old$fit$handle` is <pointer: 0x126e4b820>
+#> `old$fit$handle` is <pointer: 0x132393ef0>
 #> `new$fit$handle` is <pointer: 0x0>
 #> 
 #> `old$fit$handle` is attr(,"class")
 #> `new$fit$handle` is attr(,"class")
 #> 
 #> `old$fit$handle` is [1] "xgb.Booster.handle"
 #> `new$fit$handle` is [1] "xgb.Booster.handle"
+#> 
```

Comment:
Unrelated to the changes in this PR, but:

I'm surprised by the additions following this line. I was able to replicate with `devtools::build_readme()`, but it went away when clicking the Knit button. I overlooked some difference in environment when I filed #31.